src/Common/src/Interop/Windows/perfcounter/Interop.FormatFromRawValue.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class PerfCounter
+    {
+#pragma warning disable BCL0015 // Invalid Pinvoke call
```

Comment:
If you or Dan added an own file to disable the analyzer you can remove the warning disable here